recipe/meta.yaml

```diff
@@ -57,7 +57,7 @@ requirements:
     - oss2 ==2.6.1
     - paramiko >=2.5.0
     - pyarrow ==0.14.0
-    - tqdm >=4.38.0
+    - tqdm >=4.40.0
```

Comment:
Might makes sense to mirror requirement to not use tqdm of v.5 and higher https://github.com/iterative/dvc/blob/master/setup.py#L80
```
tqdm >=4.40.0,<5
```

Reply:
@MaxRis Thanks! Adjusted.